templates/distribution/manifests/networking/kustomization.yaml.tpl

```
@@ -50,6 +55,9 @@ patches:
       name: cilium-operator
       namespace: kube-system
     path: patches/cilium/cilium-operator-tolerations.yaml
+    {{- if not (.spec | digAny "kubernetes" "advanced" "kubeProxy" "enabled" true) }}
```

maybe add here the same comment as [cilium-config.env.tpl#R13](https://github.com/sighupio/distribution/pull/459/files#diff-1452617a5c664227166667560563a1f2a1f3f368c217300204953125f2ca44b4R13), just to clarify that we need this condition as it's written
```suggestion
    {{- /* We assume that kubeProxy is enabled by default */}}
    {{- /* The `digAny` condition needs to be exactly as written below to properly check if the field has been populated */}}
    {{- if not (.spec | digAny "kubernetes" "advanced" "kubeProxy" "enabled" true) }}
```